layers/meta-resin-beaglebone/recipes-core/images/resin-image-flasher.bbappend

```diff
@@ -1 +1,3 @@
 include resin-image.inc
+
+RESIN_BOOT_PARTITION_FILES_append_beaglebone = " uEnv.txt_internal: "
```

Comment:
That space at the end was not added for the _append. It was added for clarity on the RESIN_BOOT_PARTITION_FILES variable itself. Changed as per the comment however